and-cli-deploy-azure-web-app.js

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env node
+require("./command-runner").run(async () => {
+    // -----------------------------------------------------------------------------------------
+    // #region Imports
+    // -----------------------------------------------------------------------------------------
+
+    const azure         = require("./_modules/azure");
+    const dotnetPath    = require("./_modules/dotnet-path");
+    const echo          = require("./_modules/echo");
+    const program       = require("commander");
+    const shell         = require("shelljs");
+
+    // #endregion Imports
+
+    // -----------------------------------------------------------------------------------------
+    // #region Variables
+    // -----------------------------------------------------------------------------------------
+
+    let   appName             = null;
+    let   branch              = null;
+    let   clientId            = null;
+    let   force               = false;
+    const pythonInstallerUrl  = "https://www.python.org/ftp/python/3.7.4/python-3.7.4-amd64.exe";
```

Comment:
Thought: Maybe this could be a shared constants file _or something_ shared between the azure related CLI files